src/plugins/shared_ux/public/components/empty_state/no_data_views/documentation_link.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  EuiDescriptionList,
+  EuiDescriptionListDescription,
+  EuiDescriptionListTitle,
+  EuiLink,
+  useEuiTheme,
+} from '@elastic/eui';
+import React from 'react';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+import { css } from '@emotion/css';
+
+interface Props {
+  documentationUrl: string;
+}
+
+export function DocumentationLink({ documentationUrl }: Props) {
+  const { euiTheme } = useEuiTheme();
+  const { colors, size, border } = euiTheme;
+  const titleCSS = css`
+    width: auto !important;
+  `;
+  const footerCSS = css`
```

Comment:
EUI has actually beefed up it's [EuiEmptyPrompt](https://elastic.github.io/eui/#/display/empty-prompt) component to enable this type of layout which should allow removal of _most_ of these custom styles. Would you like me to attempt to convert these components?

Reply:
Would be awesome, thanx!

Reply:
PR4U: https://github.com/majagrubic/kibana/pull/6